publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/CuratorPermissionStrategy.java

```diff
@@ -12,6 +13,10 @@ public boolean hasPermission(RequestInfo requestInfo, Publication publication) {
         if (!userIsFromSameInstitutionAsPublication(requestInfo, publication)) {
             return false;
         }
+        if (isDegree(publication)
+            && userIsFromSameInstitutionAsPublication(requestInfo, publication)) {
```

Comment:
trenger kanskje ikke denne linja siden det allerede er sjekket over på linje 13?

Reply:
Lagt inn test og fjernet ekstra kallet. La til det unødvendige kallet for at hvis man fjerner if-statementen over så ville fortsatt if-sjekken under være gyldig.